samples/basic-component/README.md

```diff
@@ -0,0 +1,5 @@
+This sample demonstrates how to create a basic Geocortex Web layout component with an associated model. To learn more about creating components using the SDK, see our documentation in the [Developer Center](https://developers.geocortex.com/docs/web/sdk-components-overview).
```

Comment:
@eric-parsons I've added a readme to each sample. If you don't mind can you review the content to make sure it jives?

Reply:
Looks good!